composer.json

```diff
@@ -14,10 +14,13 @@
     }],
     "require": {
         "php": ">=5.3.0",
-        "symfony/framework-bundle" : ">=2.0"
+        "symfony/framework-bundle" : ">=2.0",
+        "doctrine/cache": "~1.3",
+        "guzzlehttp/guzzle": ">=4.0"
```

Comment:
:-1: For this. Guzzle could be use for project with other incompatible version. Do we really need this complete library just for download an avatar? A lighter library like buzz could be enough.